### DISCUSSION TITLE
Plurals working for English, half-working for Croatian translation, why?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [i18next/i18next#1754] [Q&A] [answered]
## Intro
I have 2 languages, Croatian and English, on the English side all plurals are working as expected, on the Croatian side, half of the translations work (even when the same key is used!)

## Version
(I am using next-18next, but it's using this library under the hood)
  "next-i18next": "^10.2.0",

## Code used 

```
    const { t } = useTranslation();
    
        const options = [
        { count: 0, duration: "day" },
        { count: 1, duration: "week" },
        { count: 2, duration: "week" }, // this doesn't work in croatian
        { count: 3, duration: "week" }, // nor this
        { count: 1, duration: "month" },  
        { count: 3, duration: "month" }, // nor this
        { count: 6, duration: "month" },
    ];

             <select>
                    {options.map((it) => (
                        <option
                            key={it.count + it.duration}
                            label={t(
                                "common:fields.items." + it.duration, //duration is month/week/day
                                { count: it.count },
                            )}
                            value={it.count + " " + it.duration}
                            className="w-full py-2"
                        ></option>
                    ))}
                </select>
```

And my translations are as follows

```
//hr/common.json
      "items": {
        "day_one": "{{count}} dan",
        "day_other": "{{count}} dana",
        "month_one": "{{count}} mjesec",
        "month_other": "{{count}} mjeseci",
        "week_other": "{{count}} tjedna",
        "week_one": "{{count}} tjedan"
      },
   
   
   //en/common.json
         "items": {
        "day_one": "{{count}} day",
        "day_other": "{{count}} days",
        "month_one": "{{count}} month",
        "month_other": "{{count}} months",
        "week_other": "{{count}} weeks",
        "week_one": "{{count}} week"
      },
```
   

## Screenshots
Here are the two translations 
![en](https://user-images.githubusercontent.com/50072027/163672682-2ac36b85-f657-4665-85b1-6ce7e329bcbb.png)
![croatian1](https://user-images.githubusercontent.com/50072027/163672683-c10e4198-3323-4687-bd8e-6a2a0f03b092.png)

## Specs
 - Device: Ubuntu 20.04
 - Browser: Chromium Version 100.0.4896.127 (Official Build) snap (64-bit)

As you can see, for some reason English works perfectly, the Croatian works but half half. The weirdest part is that Croatian does indeed work for example months, 1 and 6 months work, but 3 doesn't. And 3 is plural just the same as 6, so why it's not working is even weirder. 

I am not raising an issue as it may be something that I am unaware of

**Comment by jamuhl** [comment, marked as answer]
simple as checking https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#hr

there is not only `one` and `other` but also `few` for Croatian
